.gitlab-ci.yml

```diff
@@ -4,6 +4,11 @@ variables:
   BUILD_STABLE_REGISTRY: '486234852809.dkr.ecr.us-east-1.amazonaws.com'
   CI_IMAGE: '$BUILD_STABLE_REGISTRY/ci/$APP:$CURRENT_CI_IMAGE'
 
+cache:
+  key: '$CI_JOB_NAME-$CI_COMMIT_REF_SLUG'
+  paths:
+    - node_modules/
```

Comment:
since most of the jobs are started in parallel, this cache is not leverage as much as it could be.
what about add a job to build this cache first?

Reply:
Good point ! Sure